### DISCUSSION TITLE
How can I sync to Github?

### DISCUSSION THREAD

**Discussion by progmotic** [specstoryai/getspecstory#10] [Q&A] [answered]
I'm using the Cursor app in Linux Mint.
Where is history stored locally?
How can I sync SpecStory to Github?
Thank you!

**Comment by belucid** [comment, marked as answer]
Hey, so I assume you have the SpecStory extension [http://get.specstory.com/cursor-extension](http://get.specstory.com/cursor-extension) installed in Cursor. And then you can use the command palette (CMD-Shift-P) and type "SpecStory" and select the "Save Composer and Chat History". That opens a picker with all your workspace's chats and composers, and you can keep all of them selected, or select just some of them. 

That opens a tab with a markdown of the history of all the selected chats and composers. You can then save this tab as a file as normal, and check the file into GitHub.

We're going to change this in future versions of the extension so that you can have it continual capture the history to a file, that you keep checked into GitHub if you like (or .gitignore if you don't).

Let me know if that helps!

**Reply by progmotic** [reply]
That makes perfect sense.
Yes, the continual capture would be most useful.
Thanks!
